packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/testIgnoreAllOfHistoryList.st

```diff
@@ -0,0 +1,5 @@
+testing-functionality
+testIgnoreAllOfHistoryList
+	
+	self changesBrowser ignoreAllOfHistoryList.
+	self changesBrowser historyList do: [:each | self assertNoChangeListItemSelectedFor: each]
```

Comment:
Cascade doesn't seem to work here because in the second cascade branch we use a method chain. We believe cascades only support one method call in each branch.